frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,7 +128,8 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            settings_override = getattr(spider, 'frontier_settings', {})
```

Comment:
@sibiryakov in the PR I closed you said that you preferred the way I actuall did :-/